magda-int-test/src/test/scala/au/csiro/data61/magda/api/BaseSearchApiSpec.scala

```diff
@@ -173,8 +170,8 @@ trait BaseSearchApiSpec extends BaseApiSpec with RegistryConverters with Protoco
   }
 
   override def afterEach() {
+//    cleanUpIndexes()
     super.afterEach()
-    cleanUpIndexes()
```

Comment:
Just noticed this too - why get rid of the cleanup? Is it slowing things down?

Reply:
No index cache anymore.